src/main/java/com/evolveum/polygon/connector/msgraphapi/UserProcessing.java

```diff
@@ -1087,6 +1162,10 @@ protected boolean handleJSONObject(OperationOptions options, JSONObject user, Re
             user = saturateRoleMembership(user);
         }
 
+        if (!Boolean.TRUE.equals(options.getAllowPartialAttributeValues())) {
```

Comment:
this seems an appropriate place to also add a check for the "ATTRS_TO_GET" operation options regarding the 'photo' parameter so there is no additional roundtrip.

Reply:
done